tests/helpers/doc-links.ts

```diff
@@ -0,0 +1,19 @@
+import { Page } from "@playwright/test";
+import { expect } from "../fixtures/lxd-test";
+
+export const validateDocumentationLink = async (
+  page: Page,
+  linkText: string,
+  link: string,
+  setting = "",
+) => {
+  if (setting) {
+    await page
+      .locator("css=tr", { hasText: setting })
+      .getByRole("button")
+      .click();
+  }
```

Comment:
Probably better structured to split this into a dedicated `editServerSetting` function (or find a better suiting name). Then this function doesn't need an optional parameter and can focus on the doc link validation.